tests/integration/420.command.init.test.js

```diff
@@ -491,7 +473,7 @@ test('netlify init existing Next.js site with existing plugins', async () => {
           provider: 'manual',
           repo_branch: 'main',
           repo_path: 'git@github.com:owner/repo.git',
-          functions_dir: functions,
+          functions_dir: 'netlify/functions',
```

Comment:
Is `defaultFunctionsDirectory` in scope here?

Reply:
Oops! It is. I missed this one. Updated in https://github.com/netlify/cli/pull/4979/commits/a046cc59cd5ac6eeb7f42d84945ca012fba52911.